lib/pushmi_pullyu/swift_depositer.rb

```diff
@@ -23,27 +23,30 @@ def deposit_file(file_name, swift_container)
 
     era_container = swift_connection.container(swift_container)
 
-    deposited_file = if era_container.object_exists?(file_base_name)
-                       era_container.object(file_base_name)
-                     else
-                       era_container.create_object(file_base_name)
-                     end
+    object_exists = era_container.object_exists?(file_base_name)
 
     # Add swift metadata with in accordance to AIP spec:
     # https://docs.google.com/document/d/154BqhDPAdGW-I9enrqLpBYbhkF9exX9lV3kMaijuwPg/edit#
-    metadata = {
+    swift_metadata = {
```

Comment:
Calling this swift_metadata is redundant. We know it's swift metadata as we are in the swift_depositer class and interacting with swift. I named it metadata as this is the same naming scheme used in Openstack Ruby gem: https://github.com/ruby-openstack/ruby-openstack/blob/master/lib/openstack/swift/storage_object.rb#L11